firebird/__init__.py

```diff
@@ -47,6 +48,32 @@
 logger.setLevel(LOG_LEVEL)
 
 
+def sparkcon():
```

Comment:
actually, no, this isn't a SparkConfiguration object, its a SparkContext, which is why i went with 'sparkcon'